src/pages/billing/List.js

```diff
@@ -171,157 +170,147 @@ const NewBillingProjectModal = ({ onSuccess, onDismiss, billingAccounts, loadAcc
   ])
 }
 
-export const BillingList = _.flow(
-  Utils.withCancellationSignal,
-  Utils.connectStore(authStore, 'authState')
-)(class BillingList extends Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      billingProjects: null,
-      creatingBillingProject: false,
-      billingAccounts: null,
-      isOwner: false,
-      ...StateHistory.get()
-    }
-  }
+export const BillingList = ({ queryParams: { selectedName } }) => {
+  // Hooks
+  const [billingProjects, setBillingProjects] = useState(StateHistory.get().billingProjects || null)
```

Comment:
Why do we use stateHistory here but nothing else compared to the old code which looks like it relied on stateHistory for setting a bit of state?

Reply:
This is exactly the kind of clarification I like from converting to hooks: we were merging the whole thing into state, but down below it turns out that the billing project list is the only thing actually being saved into state at all!